## 15.7 Example 2: Dementia

Let see a second example with several variables to see how to interpret the results. This time, our interest lies in modeling the probability of developing dementia during the follow-up according to sex, age at inclusion and bmi at inclusion.


$$\mathrm{logit}(\pi_i) = \beta_0 + \beta_1sex_i + \beta_2 age_i + \beta_3 bmi_i$$

This model can be estimated in `R`using the `glm` function

In [9]:
dementia2 <- glm(dementia ~ sex + age + bmi, data = dementia, family = binomial(link="logit"))

As usually, let us print the results that have been stored in the `R`object `dementia2`.

In [10]:
summary(dementia2)


Call:
glm(formula = dementia ~ sex + age + bmi, family = binomial(link = "logit"), 
    data = dementia)

Deviance Residuals: 
    Min       1Q   Median       3Q      Max  
-1.1067  -0.1959  -0.1134  -0.0732   3.6917  

Coefficients:
             Estimate Std. Error z value Pr(>|z|)    
(Intercept) -9.783837   0.152138 -64.309  < 2e-16 ***
sex          0.306798   0.033773   9.084  < 2e-16 ***
age          0.098682   0.001413  69.826  < 2e-16 ***
bmi         -0.025619   0.003596  -7.124 1.05e-12 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

(Dispersion parameter for binomial family taken to be 1)

    Null deviance: 38333  on 199999  degrees of freedom
Residual deviance: 31732  on 199996  degrees of freedom
AIC: 31740

Number of Fisher Scoring iterations: 8


We compute the exponential of the estimated coefficients and will interpret each.

In [11]:
exp(coefficients(dementia2))

(Intercept)          sex          age          bmi 
5.635516e-05 1.359066e+00 1.103716e+00 9.747061e-01

Contrarily to the previous model, here we do not interpret the intercept. Indeed, if we were doing so, it would be the prevalence of dementia among men with age and bmi at inclusion of 0 which obviously does not make any clinical sense. We can interpret the other parameters

* *sex*: there is an association between sex and the dementia status and the odds of dementia are multiplied by $1.36$ for women compared to men having same age and bmi at inclusion
* *age*: there is an association between age and the dementia status and the odds of dementia are multiplied by $1.10$ when age increases by $1$ year assuming sex and bmi at inclusion remain unchanged
* *bmi*: there is an association between bmi and the dementia status and the odds of dementia are multiplied by $0.97$ when bmi increases by $1$ unit assuming sex and age at inclusion remain unchanged

By including these variables into this new model, we only added two parameters but we can see that the deviance decreased slightly. It was $38143$ on the first simple model and now equals $31732$. The quality of this new model is confirmed by the AIC which was $38147$ for the previous model and decreases to $31740$ for the new model.